src/services/sessionService.ts

```diff
@@ -0,0 +1,119 @@
+/* eslint-disable class-methods-use-this */
+import { JWTPayload, createRemoteJWKSet, jwtVerify } from 'jose';
+
+import { Request } from 'express';
+import User from '../entities/user.js';
+import { Assert } from '../heplers/index.js';
+
+export interface SessionInterface {
+  validateShortSessionValue(shortSession: string): Promise<object | null>;
+}
+
+interface DecodedValue extends JWTPayload {
+  name?: string;
+  email?: string;
+  phone_number?: string;
+  sub?: string;
+}
+
+interface RequestWithCookies extends Request {
+  cookies: {
+    [key: string]: string;
+  };
+}
+
+const MIN_TOKEN_LENGTH = 10;
+
+class Session implements SessionInterface {
+  private shortSessionCookieName: string;
+
```

Comment:
@sam-corbado Maybe less blank lines between class properties?

Reply:
It's a standard TS class formatting practice. See [explanation here](https://eslint.style/rules/default/lines-between-class-members).